internal/resolvers/wikipedia/load_test.go

```diff
@@ -0,0 +1,137 @@
+package wikipedia
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+
+	"github.com/Chatterino/api/pkg/utils"
+	qt "github.com/frankban/quicktest"
+	"github.com/go-chi/chi/v5"
+)
+
+func testLoadAndUnescape(c *qt.C, locale, page string) (cleanTooltip string) {
+	const urlFormat = "https://%s.wikipedia.org/wiki/%s"
+	urlString := fmt.Sprintf(urlFormat, locale, page)
+	iret, _, err := load(urlString, nil)
+
+	c.Assert(err, qt.IsNil)
+	c.Assert(iret, qt.Not(qt.IsNil))
+
+	response := iret.(response)
+
+	c.Assert(response, qt.Not(qt.IsNil))
+	c.Assert(response.err, qt.IsNil)
+
+	c.Assert(response.resolverResponse, qt.Not(qt.IsNil))
+
+	cleanTooltip, unescapeErr := url.PathUnescape(response.resolverResponse.Tooltip)
+	c.Assert(unescapeErr, qt.IsNil)
+
+	return cleanTooltip
+}
+
+var (
+	wikiData = map[string]*wikipediaAPIResponse{}
+)
```

Comment:
I'd suggest moving those lines above `testLoadAndUnescape` function to keep the file structure: package, imports, const's, var's, functions